guides/common/modules/ref_storage-requirements.adoc

```diff
@@ -31,9 +28,39 @@ ifdef::satellite[]
 |/var/lib/containers if using {insights-iop} |10 GB |15 GB
 endif::[]
 |====
++
+These values are based on expected use case scenarios and can vary according to individual environments.
+ifdef::katello,satellite[]
+The runtime size was measured with {EL} 7, 8, and 9 repositories synchronized.
```

Comment:
todo for future us: we should update the list of OSes here (and check their sizes)

Reply:
Future us might appreciate an issue so I created one: https://github.com/theforeman/foreman-documentation/issues/4195